apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -77,18 +79,25 @@ class ContributorRepository implements Repository {
 			.groupBy("contributors.id")
 			.withGraphFetched("gitEmails");
 
+		if (contributorName) {
+			query.whereILike("contributors.name", `%${contributorName}%`);
+		}
+
+		const { results, total } = await query.page(page, pageSize).execute();
+
 		return {
-			items: results.map((contributor) => {
-				return ContributorEntity.initialize(contributor);
-			}),
+			items: results.map((contributor) =>
+				ContributorEntity.initialize(contributor),
+			),
 			totalItems: total,
 		};
 	}
 
 	public async findAllByProjectId(
 		projectId: number,
+		contributorName?: string,
 	): Promise<{ items: ContributorEntity[] }> {
```

Comment:
It may be not clear, that second parameter is `contributorName` outside of this repository
```suggestion
	public async findAllByProjectId({
		projectId,
		contributorName,
	}: {
		projectId: number,
		contributorName?: string,
	}): Promise<{ items: ContributorEntity[] }> {
```
Same for `findAllWithoutPagination` and service's methods

Reply:
You forget about similar service's methods